exporter/collector/integrationtest/testcases/testcases_metrics.go

```diff
@@ -52,6 +52,13 @@ var MetricsTestCases = []TestCase{
 		OTLPInputFixturePath: "testdata/fixtures/metrics/basic_prometheus_metrics.json",
 		ExpectFixturePath:    "testdata/fixtures/metrics/basic_prometheus_metrics_expect.json",
 	},
+	{
+		Name:                 "Basic Prometheus metrics with stale data point",
+		OTLPInputFixturePath: "testdata/fixtures/metrics/basic_prometheus_metrics_stale.json",
+		ExpectFixturePath:    "testdata/fixtures/metrics/basic_prometheus_metrics_stale_expect.json",
+		CompareFixturePath:   "testdata/fixtures/metrics/basic_prometheus_metrics_expect.json",
+		SkipForSDK:           true,
```

Comment:
@dashpole should our sdk exporter drop stale points too? This failed for sdk and I didn't see anything handling stale points there

Reply:
The SDK shouldn't have a notion of stale points.  I.e. it shouldn't be possible to get those when coming from the SDK.